docs/admin/install/prepare.rst

```diff
@@ -33,7 +33,7 @@ Follow the linked instructions to `verify the ISO <https://www.qubes-os.org/secu
 .. code-block:: sh
 
   gpg --keyserver-options no-self-sigs-only,no-import-clean --fetch-keys https://keys.qubes-os.org/keys/qubes-release-4.2-signing-key.asc
-  gpg -v --verify Qubes-R4.2.2-x86_64.iso.DIGESTS
+  gpg -v --verify Qubes-R4.2.2-x86_64.iso.DIGESTS && sha256sum -c Qubes-R4.2.2-x86_64.iso.DIGESTS || echo FAILED VERIFICATION
```

Comment:
I don't think we need the separate echo? It already outputs "FAILED" if it's invalid:

```
$ sha256sum -c Qubes-R4.2.2-x86_64.iso.DIGESTS.bad 
Qubes-R4.2.2-x86_64.iso: FAILED
sha256sum: WARNING: 20 lines are improperly formatted
sha256sum: WARNING: 1 computed checksum did NOT match
```

So my suggestion is:
```suggestion
  gpg -v --verify Qubes-R4.2.2-x86_64.iso.DIGESTS
  sha256sum -c Qubes-R4.2.2-x86_64.iso.DIGESTS
```